.travis.yml

```diff
@@ -0,0 +1,14 @@
+sudo: false
+language: perl
+perl:
+    - "5.22"
+    - "5.20"
+    - "5.18"
+    - "5.16"
+    - "5.14"
+    - "5.12"
+    - "5.10"
```

Comment:
Why not? This is open-source module, let it be tested on different perl versions